src/Service/PagePurger.php

```diff
@@ -43,6 +51,41 @@ public function purge( Page $page ) {
 	}
 
 	/**
+	 * @since 0.7
+	 *
+	 * @brief Purge multiple pages
+	 *
+	 * Purges all the pages of the Pages object
+	 * by submitting a 'purge' action to the mediawiki
+	 * api with the parameter 'pageids' set to be the
+	 * pages ids in multiple-value seperation.
+	 *
+	 * @param Pages $pages the pages that are going to be purged
+	 *
+	 * @return bool
```

Comment:
Does the API response list the pages that were successfully purged?
If so then yes perhaps returning a Pages object here would make sense!